test/testmathlib.cpp

```diff
@@ -312,6 +313,11 @@ class TestMathLib : public TestFixture {
         ASSERT_EQUALS_DOUBLE(0.0   , MathLib::toDoubleNumber("-0.0"));
         ASSERT_EQUALS_DOUBLE(0.0   , MathLib::toDoubleNumber("+0.0"));
 
+        // Ticket #6795
+        std::setlocale(LC_NUMERIC, "de_DE");
```

Comment:
The locale names on the systems are not unique. For example on my system (Ubuntu 12.04) the locale "de_DE" does not exist. The corresponding locale there is named "de_DE.utf8".
If the the locale name does not exist on the system the std::setlocale function returns nullptr and the call has no effect.
In this case your test is not significant.
The question is which locale names should be tried in the test case in order to get a good coverage on the different systems.